Solution1/BusinessLayer/Models/CityOption.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BusinessLayer.Models
```

Comment:
there aren't models in BL, you can call CityOptionDTO

Reply:
+